contentcuration/contentcuration/migrations/0149_auto_20240612_1847.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.24 on 2024-06-12 18:47
```

Comment:
Seems like these were generated while pointing at your local version of le-utils? Would be good to include the upgrade of le-utils in this PR as well.